<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AineistoDialog</class>
 <widget class="QDialog" name="AineistoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kirjanpitoaineiston muodostaminen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Toiminto muodostaa koko tilikaudesta yhden ison pdf-tiedoston. Tiedoston muodostaminen saattaa epäonnistua, jos tietokoneessa ei ole riittävästi muistia ja kirjanpidossa on paljon tositteita.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tilikausi</string>
       </property>
       <property name="buddy">
        <cstring>tilikausiCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="tilikausiCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tiedostonnimi</string>
       </property>
       <property name="buddy">
        <cstring>tiedostoEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="tiedostoEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tiedostoNappi">
         <property name="text">
          <string>Vaihda</string>
         </property>
         <property name="icon">
          <iconset resource="../pic/pic.qrc">
           <normaloff>:/pic/avaatiedosto.png</normaloff>:/pic/avaatiedosto.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Kieli</string>
       </property>
       <property name="buddy">
        <cstring>kieliCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KieliCombo" name="kieliCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="raportitGroup">
         <property name="title">
          <string>Raportit</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="taseCheck">
            <property name="text">
             <string>Tase</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="tuloslaskelmaCheck">
            <property name="text">
             <string>Tuloslaskelma</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="erittelyCheck">
            <property name="text">
             <string>Tase-erittely</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="paivakirjaCheck">
            <property name="text">
             <string>Päiväkirja</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="paakirjaCheck">
            <property name="text">
             <string>Pääkirja</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="myyntilaskutCheck">
            <property name="text">
             <string>Avoimet myyntilaskut</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ostolaskutCheck">
            <property name="text">
             <string>Avoimet ostolaskut</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="tositeluetteloCheck">
            <property name="text">
             <string>Tositeluettelo</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="tililuetteloGroup">
         <property name="title">
          <string>Tililuettelo</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="kaikkitlRadio">
            <property name="text">
             <string>Kaikki tilit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="laajuustlRadio">
            <property name="text">
             <string>Tilikartan laajuuden mukaan</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="kirjauksettlRadio">
            <property name="text">
             <string>Tilit joilla kirjauksia</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="tiedotGrop">
         <property name="title">
          <string>Tositteiden tiedot</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="muistiinpanotCheck">
            <property name="text">
             <string>Muistiinpanot</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="tilioinnitCheck">
            <property name="text">
             <string>Täydet tiliöinnit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kaikkiCheck">
            <property name="text">
             <string>Kaikkien tositteiden tiedot</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="liitteetGroup">
         <property name="title">
          <string>Liitteet</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Kuvanlaatu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LaatuSlider" name="dpiSlider" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LaatuSlider</class>
   <extends>QWidget</extends>
   <header location="global">arkistoija/laatuslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KieliCombo</class>
   <extends>QComboBox</extends>
   <header location="global">kieli/kielicombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tilikausiCombo</tabstop>
  <tabstop>tiedostoEdit</tabstop>
  <tabstop>kieliCombo</tabstop>
  <tabstop>tiedostoNappi</tabstop>
  <tabstop>taseCheck</tabstop>
  <tabstop>tuloslaskelmaCheck</tabstop>
  <tabstop>erittelyCheck</tabstop>
  <tabstop>paivakirjaCheck</tabstop>
  <tabstop>paakirjaCheck</tabstop>
  <tabstop>myyntilaskutCheck</tabstop>
  <tabstop>ostolaskutCheck</tabstop>
  <tabstop>tositeluetteloCheck</tabstop>
  <tabstop>kaikkitlRadio</tabstop>
  <tabstop>laajuustlRadio</tabstop>
  <tabstop>kirjauksettlRadio</tabstop>
  <tabstop>muistiinpanotCheck</tabstop>
  <tabstop>tilioinnitCheck</tabstop>
  <tabstop>kaikkiCheck</tabstop>
 </tabstops>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AineistoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>747</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AineistoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>753</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
